imports/ui/proposals/Proposal.jsx

```diff
@@ -270,6 +270,22 @@ export default class Proposal extends Component{
         </Result>
     }
 
+    renderValidatorsWithoutVote() {
+        return <Result className="tally-result-detail" pose={this.state.open ? 'open' : 'closed'}>
+            <Card className='tally-result-table'>
+                {this.props.proposal.validatorsDidNotVote.list.map((vote, i) =>
```

Comment:
@MonikaCat can any of the objects here be undefined?

Reply:
Ok right, the value becomes undefined when it can't read the data from the db. I have updated the code to handle it.